mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -161,6 +161,7 @@ func IsMesheryRunning(currPlatform string) (bool, error) {
 			deploymentList, err := deploymentInterface.List(context.TODO(), metav1.ListOptions{})
 
 			if err != nil {
+				err = errors.New("Meshery Server unavailable at http://localhost:13021. Is your Meshery Server up and running?")
```

Comment:
```suggestion
				err = errors.New("Meshery Server unavailable at http://localhost:13021. Is your Meshery Server up and running?\n\nUse `mesheryctl system context` to confirm or switch contexts.")
```

Reply:
Could we take advantage to upgrade to a Meshkit error 

https://docs.meshery.io/project/contributing/contributing-error ?